topydo/lib/Config.py

```diff
@@ -186,6 +193,49 @@ def hidden_tags(self):
         hidden_tags = self.cp.get('ls', 'hide_tags')
         return [] if hidden_tags == '' else hidden_tags.split(',')
 
+    def priority_colors(self):
+        """ Returns a dict with priorities as keys and color numbers as value. """
+        pri_colors_str = self.cp.get('colorscheme', 'priority_colors')
+
+        def _str_to_dict(p_string):
+            pri_colors_dict = dict()
+            for pri_color in pri_colors_str.split(','):
```

Comment:
Looking at it again, I wonder if falling back to `NEUTRAL_COLOR` is appropriate for any color setting. If, for instance, the project color is missing, I think it's better to use the priority color of that item (if configured and applicable).

```ini
[colorscheme]
project_color =
```

```
(C) This item would be shown +blue by default
                             ^^^^^
                             white
```

Instead, I would expect `+blue` to turn up... well... blue.

Returning to your question for the priorities, that would mean that all todo items will appear neutrally colored when they upgrade topydo, since they don't have a configuration file where `priority_colors` is defined. I think we should always use sane defaults reflecting the current behavior (topydo < 0.4).